src/documents/ComponentConfigurationOptions.md

```diff
@@ -332,86 +321,76 @@ fluid.defaults("component.name", {
-
-See also: [Component Grades](ComponentGrades.md)
-
-### events ###
+### `modelListeners` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object containing key/value pairs that define the events the component will fire: the keys are the event names, the values define the type of the event (see <a href="InfusionEventSystem.md">Infusion Event System</a> for information on the different event types).</td>
-  </tr>
-  <tr>
-    <th>Notes</th>
-    <td>The Framework will create event firers for the listed events. It is the responsibility of the component to fire the events at the appropriate times.</td>
+    <td>A record defining a set of functions wishing to be notified of changes to the `model`</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
-<code>fluid.defaults("component.name", {
-    events: {
-        onSave: "preventable",
-        onReady: null
-    },
-    ...
-});</code>
+fluid.defaults("fluid.tests.allChangeRecorder", {
+    gradeNames: "fluid.tests.changeRecorder",
+    modelListeners: {
+        "": "{that}.record({change}.path, {change}.value, {change}.oldValue)"
+    }
+});
 </pre></td>
   </tr>
-  <tr>
+   <tr>
     <th>See also</th>
-    <td><a href="InfusionEventSystem.md">Infusion Event System</a></td>
+    <td><a href="ChangeApplierAPI.md#model-listener-declaration">Model Listeners</a></td>
   </tr>
 </table>
 
-### listeners ###
+### `modelRelay` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object defining listener functions for the events supported by a component.</td>
-  </tr>
-  <tr>
-    <th>Notes</th>
-    <td>Both component developers and integrators can define listeners for events.
-<a href="Invokers.md">Invokers</a> and <a href="ExpansionOfComponentOptions.md">Expanders</a> can be used as listeners here. Note that as well as being a simple string holding the name of an event on this component, a listener key may also be a full <a href="IoCReferences.md">IoC Reference</a> to any other event held in the component tree (for example <code>"{parentComponent}.events.parentEvent"</code>. As well as being a simple function name, a the value associated with the key may be a <a href="InfusionEventSystem.md">Listener Record</a> or else follow the syntax of an invoker indicating that the registered listener receives a different signature from the one that the event has fired (see <a href="EventInjectionAndBoiling.md">Event injection and boiling</a>).</td>
+    <td>A set of rules or constraints linking values held in this model to those elsewhere in the component tree (or to other values within this model)</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
-<code>fluid.defaults("component.name", {
-    events: {
-        onSave: "preventable",
-        onReady: null
-    },
-    listeners: {
-        onSave: "component.name.saveValidatorFn"
+fluid.defaults("examples.volumeModelRelay", {
```

Comment:
It looks the same as all the other examples to me: https://github.com/amb26/infusion-docs/blob/FLUID-5249/src/documents/ComponentConfigurationOptions.md